scripts/index.mjs

```diff
@@ -13,25 +13,31 @@
 
 
 
-const fs = require('fs-extra');
-const os = require('os');
-const _ = require('lodash');
-const path = require('path');
-const cp = require('child_process');
-const inquirer = require('inquirer');
-const fuzzy = require('fuzzy');
-const chalk = require('chalk');
-const moment = require('moment');
-const Checkbox = require('inquirer-checkbox-plus-prompt');
-const commander = require('commander');
-const killPort = require('kill-port');
-
-// const rollup = require('rollup');
-// const loadConfigFile = require('rollup/loadConfigFile');
+import chalk from 'chalk';
+import cp from 'child_process';
+import * as commander from 'commander';
+import fs from 'fs-extra';
+import fuzzy from 'fuzzy';
+import inquirer from 'inquirer';
+import Checkbox from 'inquirer-checkbox-plus-prompt';
+import killPort from 'kill-port';
+import _ from 'lodash';
+import moment from 'moment';
+import path from 'node:path';
+import process from 'node:process';
+import { fileURLToPath } from 'node:url';
+import os from 'os';
+import { listFiles, transform as transformFiles } from './transform_files.mjs';
```

Comment:
can MJS import normal commonJS modules if needed?

Reply:
seems possible
there's also an option to import require and use that, see [SO](https://stackoverflow.com/a/56350495/9068029)

https://nodejs.org/api/esm.html#differences-between-es-modules-and-commonjs